tilelang/cuda/intrinsics/macro/wgmma_sp_macro_generator.py

```
@@ -145,89 +160,62 @@ def wgmma_ss(
-        B_ptr = retrive_ptr_from_buffer_region(B_region)
+        A_buf = A_region.buffer if isinstance(A_region, BufferRegion) else A_region
+        B_buf = B_region.buffer if isinstance(B_region, BufferRegion) else B_region
+        A_base_ptr = A_buf.access_ptr("r")
+        B_base_ptr = B_buf.access_ptr("r")
         assert is_full_region(C_region), "Fragment output C must be a full region"
         C_buf = C_region.buffer
 
         @T.macro
-        def _warp_mma(A_ptr, B_ptr, C_buf):
+        def _warp_mma(A_base_ptr, B_base_ptr, C_buf):
             tx, warp_n, warp_m = self.extract_thread_binding(thread_binding)
```

Ruff (0.15.20)

[warning] Unpacked variable `tx` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)